### PR TITLE
[Advanced LoRA v1.5] fix: gradient unscaling problem

### PR DESCRIPTION
**Pull request by sayakpaul**
# What does this PR do?

Part of https://github.com/huggingface/diffusers/issues/6552. 

@linoytsaban I couldn't find instructions to run this script in the README. Would appreciate some help in verifying it.

### CODE REVIEW

**Comment by sayakpaul** [comment]
@linoytsaban a gentle ping here.

**Comment by sayakpaul** [comment]
@linoytsaban sorry about the repeated pings here.

**Comment by linoytsaban** [comment]
sorry missed this!

**Review by linoytsaban — APPROVED** [review]
LGTM!

**Comment by sayakpaul** [comment]
Would you be able to test it as well? I don;t have the training commands for v1.5 to test.

**Comment by linoytsaban** [comment]
Apologies for the delay, tested with ⬇️ and it works good! we can ship :) Thanks a lot!
```
#!/usr/bin/env bash
!accelerate launch train_dreambooth_lora_sd15_advanced.py \
  --pretrained_model_name_or_path="runwayml/stable-diffusion-v1-5" \
  --dataset_name="./linoy_face" \
  --instance_prompt="a TOK woman"\
  --validation_prompt="watercolor painting of a TOK woman with pink hair in New York"\
  --train_text_encoder_ti\
  --output_dir="linoy_lora_v1_sd15" \
  --caption_column="prompt" \
  --mixed_precision="fp16" \
  --resolution=1024 \
  --train_batch_size=3 \
  --repeats=1\
  --report_to="wandb"\
  --gradient_accumulation_steps=1 \
  --gradient_checkpointing \
  --learning_rate=1.0 \
  --text_encoder_lr=1.0 \
  --optimizer="prodigy"\
  --lr_scheduler="constant" \
  --lr_warmup_steps=0 \
  --rank=32 \
  --max_train_steps=1000 \
  --checkpointing_steps=2000 \
  --seed="0" \
  --push_to_hub
  ```

**Comment by sayakpaul** [comment]
@linoytsaban thanks but we need to check if it can successfully resumed as well. Refer to https://github.com/huggingface/diffusers/pull/6514#issue-2073815774 for more details.

**Comment by sayakpaul** [comment]
Friendly ping here @linoytsaban
